build.gradle

```diff
@@ -400,7 +400,7 @@ subprojects {
       }
 
       violationRules {
-        minBranches = project.ext.has('diffCoverageThreshold') ? project.ext.diffCoverageThreshold : 0.6
+        minBranches = project.ext.has('diffCoverageThreshold') ? project.ext.diffCoverageThreshold : 0.33
```

Comment:
There are some modules that override this value to higher numbers, let's remove those overrides?

Reply:
```
$ grep "jacocoCoverageThreshold = " **/*.gradle
clients/da-vinci-client/build.gradle:  jacocoCoverageThreshold = 0.40
clients/venice-admin-tool/build.gradle:  jacocoCoverageThreshold = 0.04
clients/venice-client/build.gradle:  jacocoCoverageThreshold = 0.53
clients/venice-push-job/build.gradle:  jacocoCoverageThreshold = 0.39
clients/venice-thin-client/build.gradle:  jacocoCoverageThreshold = 0.44
internal/venice-client-common/build.gradle:  jacocoCoverageThreshold = 0.24
internal/venice-common/build.gradle:  jacocoCoverageThreshold = 0.26
internal/venice-consumer/build.gradle:  jacocoCoverageThreshold = 0.00
internal/venice-test-common/build.gradle:  jacocoCoverageThreshold = 0.00
services/venice-controller/build.gradle:  jacocoCoverageThreshold = 0.21
services/venice-router/build.gradle:  jacocoCoverageThreshold = 0.39
services/venice-server/build.gradle:  jacocoCoverageThreshold = 0.23
```

Reply:
I think you mean `jacocoCoverageThreshold`? If so and the module has the higher number, we just leave them as they are.

Reply:
Yes, i mean, the 0.33 is just act as a default coverage for some modules if not specified. If specified, we could just use them to keep the bar high.

Reply:
Oh yeah, I meant `jacocoCoverageThreshold`. It would seem confusing as to why some modules have a higher bar when the overall codebase is limited to 33%. But I'm okay with it if you are. Feel free to mark this as resolved

Reply:
I think your point stands. If we will have a new module, it makes more sense to set a much higher bar, e.g. 0.6 😄